subsys/bluetooth/host/adv.c

```diff
@@ -786,6 +822,7 @@ static int le_ext_adv_param_set(struct bt_le_ext_adv *adv,
 	bool dir_adv = param->peer != NULL, scannable;
 	struct net_buf *buf, *rsp;
 	int err;
+	enum adv_name_type adv_name;
```

Comment:
Isn't `name_type` a better variable name than `adv_name`? 

Alternatively rename the enum to `bt_adv_name_type` and use the full `adv_name_type` for the variable.

Reply:
Renamed to name_type.